### ISSUE TITLE
Accept `idata_kwargs` in `sampling_jax`

### ISSUE THREAD

**Issue by ricardoV94** [pymc-devs/pymc#5379]
Right now we can't pass `idata_kwargs` to `sampling_jax` (most commonly used to set `loglikelihood=False`, so that it does not compute it at the end of sampling). We should accommodate this, to bring the two APIs closer together

**Comment by samO2I1** [comment]
@ricardoV94 can I work on this issue?

**Comment by ricardoV94** [comment]
> @ricardoV94 can I work on this issue?
> 

Definitely. I assigned the issue to you. Let me know if you have any questions

**Comment by samO2I1** [comment]
@ricardoV94 Can you provide the source code link?

**Comment by twiecki** [comment]
https://github.com/pymc-devs/pymc/blob/main/pymc/sampling_jax.py#L134

**Comment by samO2I1** [comment]
@ricardoV94 @twiecki Can you tell where ###idata_kwargs is situated?

**Comment by samO2I1** [comment]
@twiecki @ricardoV94 Can you tell where **idata_kwargs** is situated?

**Comment by ricardoV94** [comment]
> @twiecki @ricardoV94 Can you tell where **idata_kwargs** is situated?

@samO2I1 you will have to look at the `sample` function in `sampling.py` to see how the`idata_kwargs` are used from there and figure out how the same behavior can be replicated in the `sample_numpyro_nuts` function found in `sampling_jax.py`

**Comment by samO2I1** [comment]
@ricardoV94 can you please explain the use of loglikelihood=False ?

**Comment by ricardoV94** [comment]
> @ricardoV94 can you please explain the use of loglikelihood=False ?

Hi @samO2I1 it seems you are not very familiar with the use of `idata_kwargs`, perhaps it's better if you pick another beginner friendly issue that is more in line with your background knowledge of the library? Let me know if you would like some help in choosing it. You can find them here: https://github.com/pymc-devs/pymc/issues?q=is%3Aopen+is%3Aissue+label%3A%22beginner+friendly%22

**Comment by ricardoV94** [comment]
@samO2I1 Apologies, I went ahead and opened a PR for this issue because it was urgent for other work.
